### DISCUSSION TITLE
After Migration applied successfully issue with JWK's(ory/Hydra v2.0.3)

### DISCUSSION THREAD

**Discussion by ashish-nikalje** [ory/hydra#3416] [Q&A]
The issue I was facing I upgraded hydra from 1.8.5 to 2.0.3
First we got the error
 ```
 The authorization server encountered an unexpected condition that prevented it from fulfilling the request. 
  Could not ensure that signing keys for hydra.openid.id-token exists. If you are running against a persistent SQL database this is most likely because your secrets.system SECRETS_SYSTEM environment variable is not set or changed. When running with an SQL database backend you need to make sure that the secret is set and stays the same%2C unless when doing key - rotation. This may also happen when you forget to run hydra migrate sql
 ```
 
we performed key rotation and after that it worked but whenever we do deploy new changes automatically all jwks getting invalid and same error getting populated

not sure why. Went through the hydra migration guide, apply upgrades, change_log but haven't found anything.

**Comment by vinckr** [comment]
Hello @ashish-nikalje 

Have you been able to resolve it? 

Since you already looked into all the documentation, hard to say what is going on without having a deeper look into your setup.
